content/blog/2025-05-08-go-client-GA.md

```diff
@@ -0,0 +1,501 @@
++++
+title= "Valkey GLIDE Go Client: Now Generally Available!"
+description = "Valkey GLIDE Go client reaches general availability. Read to learn more about the go client designed for performance and developer productivity"
+date= 2025-05-08 01:01:01
```

Comment:
Fix date

Reply:
This will get fixed when the blog is finally approved/posted.